package/utils/util-linux/Makefile

```diff
@@ -172,7 +172,7 @@ endef
 define Package/fdisk
 $(call Package/util-linux/Default)
   TITLE:=manipulate disk partition table
-  DEPENDS:= +libblkid +libsmartcols +libfdisk
+  DEPENDS:= +libblkid +libsmartcols +libfdisk +libncursesw
```

Comment:
Wasn't suggesting a patch.  Maybe just adding:

`DISABLE_NLS:=--disable-nls`

so that it's always built without NLS even if `CONFIG_BUILD_NLS=y` is set.